src/test/java/org/mule/test/oauth2/AbstractOAuthAuthorizationTestCase.java

```diff
@@ -60,11 +62,15 @@
   public static final String REFRESH_TOKEN = "cry825cyCs2O0j7tRXXVS4AXNu7hsO5wbWjcBoFFcJePy5zZwuQEevIp6hsUaywp";
   public static final String EXPIRES_IN = "3897";
   public static final String AUTHORIZE_PATH = "/authorize";
+  private static final String PROXY_CONNECTION_HEADER = "Proxy-Connection";
```

Comment:
isn't there a constant for this already in the http service?

Reply:
I haven't found it.